From a7e359d4ba345aa2a13c07f1057184e9b4e598a2 Mon Sep 17 00:00:00 2001
From: sweeaun <swee.aun.khor@intel.com>
Date: Tue, 22 Aug 2017 11:19:48 -0700
Subject: [PATCH] openssh: Fix syntax error on x32

Upstream-Status: Backport
This bug has been fixed in v_7.5 branch https://github.com/openssh/
openssh-portable/tree/V_7_5 and master branch https://github.com/
openssh/openssh-portable/tree/master.

Fix compilation error during openssh x32 build due to syntax error.

Signed-off-by: sweeaun <swee.aun.khor@intel.com>
---
 sandbox-seccomp-filter.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sandbox-seccomp-filter.c b/sandbox-seccomp-filter.c
index 3a1aedc..a8d472a 100644
--- a/sandbox-seccomp-filter.c
+++ b/sandbox-seccomp-filter.c
@@ -235,7 +235,7 @@ static const struct sock_filter preauth_insns[] = {
 	 * x86-64 syscall under some circumstances, e.g.
 	 * https://bugs.debian.org/849923
 	 */
-	SC_ALLOW(__NR_clock_gettime & ~__X32_SYSCALL_BIT);
+	SC_ALLOW(__NR_clock_gettime & ~__X32_SYSCALL_BIT),
 #endif
 
 	/* Default deny */
-- 
2.7.4

